code/modules/antagonists/ert/ert.dm

```diff
@@ -42,7 +42,8 @@
 
 /datum/antagonist/ert/deathsquad/New()
 	. = ..()
-	name_source = GLOB.commando_names
+	name_source = pick(GLOB.commando_names)
```

Comment:
How is this a fix when it unfixes something and doesnt fix anything

Reply:
This will result in the entire ERT getting single letters for names btw

Reply:
> This will result in the entire ERT getting single letters for names btw

https://streamable.com/v9paxg

Reply:
My bad, pick(pick()) seems to return the entire string (for some reason the result of the firstpick is treated as a single element list and a var at the same time), it would result in the entire ert being called the same name but it's fixed now so resolving.